setup.py

```diff
@@ -258,7 +259,12 @@ def finalize_options(self):
             include_dirs=[model_dir]),
         Extension('gensim.models.fasttext_inner',
             sources=['./gensim/models/fasttext_inner.c'],
-            include_dirs=[model_dir])
+            include_dirs=[model_dir]),
+        Extension('gensim.csparse.psparse',
+            sources=['gensim/csparse/psparse.pyx', 'gensim/csparse/CSParse/Source/cs_gaxpy.c'],
```

Comment:
1. What's about other files (~50 `.c` files in PR)?
2. Maybe we should remove all unused files (or if we shouldn't do this, why)?